spec/lib/sidekiq/form526_historical_data_exporter/form526_background_data_job_queuer_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'sidekiq/form526_historical_data_exporter/queuer'
+
+RSpec.describe Sidekiq::Form526HistoricalDataExporter::Form526BackgroundDataJobQueuer, type: :job do
+  subject { described_class }
+
+  describe 'queue_chunks' do
+    let(:user) { FactoryBot.create(:user, :loa3) }
+    let(:auth_headers) do
+      EVSS::DisabilityCompensationAuthHeaders.new(user).add_headers(EVSS::AuthHeaders.new(user).to_h)
+    end
+
+    context 'with submission after exporter start_date' do
+      let(:saved_claim) { FactoryBot.create(:va526ez) }
+      let(:submission) do
+        create(:form526_submission,
+               user_uuid: user.uuid,
+               auth_headers_json: auth_headers.to_json,
+               saved_claim_id: saved_claim.id)
+      end
+
+      let(:data_job_mock) do
+        data_job = instance_double(Sidekiq::Form526HistoricalDataExporter::Form526BackgroundDataJob)
+        allow(Sidekiq::Form526HistoricalDataExporter::Form526BackgroundDataJob).to receive(:perform_async)
+        data_job
+      end
+
+      before do
+        Flipper.enable(:disability_526_classifier)
+        submission.created_at = '2003-01-03 00:12:34'
+        submission.save
+      end
+
+      it 'gets one form526submission' do
+        start_date = '1/2/2003'
+        job_queuer = subject.new(1, 1, start_date)
+        expect(job_queuer).to receive(:data_job_wrapper).once
+        job_queuer.export
```

Comment:
` job_queuer.export`

I don't know that this actually will do anything here. It will queue the jobs, but does not actually test anything (other than that the jobs get queued without throwing an error). 

This doesn't hurt anything having this here, but just curious if there was anything you thought this was doing or if you needed/wanted to actually test anything related to what happens after the thing is queued. 

I think for what this is doing, it might not need all that much more testing than what you have here.

Reply:
I disagree. If job chunk size is 1, then export should result in `data_job_wrapper` being called for ea associated `Form526Submission`. See also `twice`
```
        it 'gets two form526submissions' do
          ...
          expect(job_queuer).to receive(:data_job_wrapper).twice
```

I agree that test coverage would be better to add something for the Exporter as well, but CI is already yelling at me about LoC

Reply:
Ah I think I see what you are saying, my bad!